unity-client/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/WebRequestController.cs

```diff
@@ -0,0 +1,58 @@
+namespace DCL
+{
+    /// <summary>
+    /// This class group all our custom WebRequests types in a single entity to ease its use.
+    /// </summary>
+    public interface IWebRequestController
+    {
+        /// <summary>
+        /// Use it for making generic requests to any url.
+        /// </summary>
+        IWebRequest webRequest { get; }
+
+        /// <summary>
+        /// Use it when you need to request an asset bundle.
+        /// </summary>
+        IWebRequestAssetBundle webRequestAB { get; }
+
+        /// <summary>
+        /// Use it when you need to request a texture.
+        /// </summary>
+        IWebRequestTexture webRequestTexture { get; }
+
+        /// <summary>
+        /// Use it when you need to request an audio.
+        /// </summary>
+        IWebRequestAudio webRequestAudio { get; }
+
+        void Initialize(
+            IWebRequest webRequest,
+            IWebRequestAssetBundle webRequestAB,
+            IWebRequestTexture webRequestTexture,
+            IWebRequestAudio webRequestAudio);
+    }
+
+    public class WebRequestController : IWebRequestController
+    {
+        public static WebRequestController i { get; private set; }
```

Comment:
I had to add the static accessor because of a lot of circular references that I got when I tried to use the `Environment` from several places where we will need to call this new web request (`Environment.platform.webRequest...`). Tomorrow I will take a look in deep in order to see how we could remove these assemby dependencies.

Reply:
As we have agreed, we are temporally going with the `WebRequestController.i` approach and we will change it when we have the `Environment` totally decoupled to avoid cyclic references.

Reply:
We are doing the same with the `QuestController`, so I agree as well that temporarily we are using this approach to be ported into environment in the future.